### ISSUE TITLE
maintainers wanted!

### ISSUE THREAD

**Issue by AndreiD** [AndreiD/UltimateAndroidAppTemplate#3]
Sadly I don't have time to maintain this project anymore.
if any of you feels like doing it, please drop me a message in this issue.

thank you!

**Comment by rhinox** [comment]
I’m interested what’s making you part ways just no time to do it?

**Comment by AndreiD** [comment]
yes man, I left "android development" some time ago. Moved to backend stuff.... so I don't know what's new in android, how to keep track with new versions coming etc...
